src/dom.js

```diff
@@ -544,3 +544,18 @@ export function escapeCssSelectorIdent(win, ident) {
   // Polyfill.
   return cssEscape(ident);
 }
+
+
+/**
+ * Returns a frame element if available.
+ * @param {!Window} win
+ * @return {?HTMLIFrameElement}
+ */
+export function getFrameElement(win) {
+  try {
+    return win.frameElement;
+  } catch (e) {
```

Comment:
Please document that that is the contract. If we only call this when we know it will work, then it is fine. Although I wonder then why we would have a try block.